indico/modules/designer/models/templates.py

```diff
@@ -130,6 +139,15 @@ def __init__(self, **kwargs):
     def owner(self):
         return self.event_new if self.event_new else self.category
 
+    @property
+    def is_not_deletable(self):
```

Comment:
I think this name is highly confusing, since `not_deletable` already exists and they hold different meanings.
I would call it `can_be_deleted` or similar. It's still not perfect, but at least it won't be confused with the column.